browser-test/src/applicant/questions/phone.test.ts

```diff
@@ -155,8 +155,8 @@ test.describe('phone question for applicant flow', () => {
 
       // Go back to question and ensure that "Review" button is also clickable
       // via Enter.
-      await page.click('a:has-text("Edit")')
-      await page.focus('a:has-text("Review")')
```

Comment:
When `SAVE_ON_ALL_ACTIONS` flag is enabled, the "Review" button is a `<button>` not a `<a>`, which is why this changed.